webpack.config.js

```diff
@@ -12,7 +12,10 @@ module.exports = {
       {test: /\.css$/, loader: ExtractTextPlugin.extract('style-loader',
         'css-loader!postcss')},
       {test: /\.less$/, loader: ExtractTextPlugin.extract('style',
-        'css!postcss!less')}
+        'css!postcss!less')},
+      // handles the bootstrap glyphicons
+      {test: /\.(png|woff|woff2|eot|ttf|svg)$/,
+        loader: 'url-loader?limit=100000&name=./statics/icons.svg'}
```

Comment:
If the `&name=./statics/icons.svg` part is used to bundle all the icons into a single SVG bundle, then:

1. It seems very confusing. We should add a comment explaining this.
2. Why do we want a single bundle?
3. I would refactor the [`output` config](https://github.com/uberVU/mocha-mugshot-reporter/blob/b0a423650400aaf72273202cb9b117232d83121f/webpack.config.js#L6-L8) and include `path: './statics'` so that we can get rid of `statics/` in both the `output.filename` and here.